docs/data/routes/docs/nextjs/introduction/why-nextjs/en.md

```diff
@@ -3,4 +3,33 @@ name: why-nextjs
 routeTemplate: ./data/component-templates/article.yml
 title: Why Next.js and JSS
 ---
-# Why Next.js and JSS
\ No newline at end of file
+# Why Next.js and JSS?
+
+[Next.js](https://nextjs.org/) is a framework built on top of **React**. It aims to help developers create production-ready applications with minimal need for configuration and boilerplate code. 
+
+The latest JSS SDK leverages many of Next.js' [features](https://nextjs.org/#features), to simplify and enhance development workflow and experience. 
+
+## Improvements over previous JSS SDKs
+
+* Development and production runtime parity, including SSR during development.
+* Fewer confusing “modes” and deployment requirements.
+* No Headless SSR Proxy needed for production or for integration with Sitecore tracking.
+* No need to deploy your application bundle to the Content Management instance for Experience Editor support.
+
+## Features
+
+- Support for Next.js static HTML export, incremental static site generation, server-side rendering, and hybrid rendering.
+- Dynamic Sitecore-based routing via Next.js pages and the Sitecore Layout Service.
+- Dynamic component rendering based on author-defined page layouts.
+- Helper components for rendering various Sitecore field types.
+- Sitecore Experience Editor integration via Next.js Preview Mode.
+- Sitecore route querying for static site generation.
+- Next.js client-side routing support for Sitecore link and rich text fields.
+- Internationalization support via Sitecore language versions and Next.js internationalized routing.
+- Component-level data fetching for Sitecore-driven components.
+- Sitecore analytics and personalization support (SSR-only).
+- Support for JSS “disconnected”/mock-Sitecore mode.
```

Comment:
@nickwesselman - should we refer to it as "Code-first approach" since we want to get away from "modes"

Reply:
Yep good feedback, changed